src/web/handlers/GestureHandlerType.ts

```diff
@@ -0,0 +1,48 @@
+import type { PointerType } from '../../PointerType';
+import type { MouseButton } from '../../handlers/gestureHandlerCommon';
+import type { State } from '../../State';
+import type { Config } from '../interfaces';
+import type EventManager from '../tools/EventManager';
+import type { GestureHandlerDelegate } from '../tools/GestureHandlerDelegate';
+import type PointerTracker from '../tools/PointerTracker';
+
+export default interface GestureHandlerType {
```

Comment:
Terrible name... How about IGestureHandler to signify the class-like contract?

Reply:
Done in [050189e](https://github.com/software-mansion/react-native-gesture-handler/pull/2783/commits/050189e29a3513052eafffd976d89a7a1daa87b6).